packages/shared/src/variants/one_color.ts

```diff
@@ -13,7 +11,7 @@ export class OneColorGo extends Baduk {
   }
 }
 
-export const oneColorGoVariant: Variant<NewBadukConfig> = {
+export const oneColorGoVariant: typeof badukVariant = {
```

Comment:
The alternative was to change to `Variant<NewBadukConfig, BadukState>`. I figured `typeof` was DRYer.